### PR TITLE
Add AssetSearch, AssetSearchForm and RootAssetSelect to Storybook

### LINKED ISSUE #6: RootAssetSelect
component RootAssetSelect

image

### PR DESCRIPTION
**Pull request by mrtraser**
Fixes #6 #20 #4 Issues

### CODE REVIEW

File: src/components/AssetSearch/AssetSearch.tsx [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+              suffix={
+                <Icon
+                  type="filter"
+                  onClick={() => this.onFilterIconClick()}
+                  style={{ opacity: 0.6, marginLeft: 8 }}
+                />
+              }
+            />
+          )}
+          {/*</Input.Group>*/}
```
Comment (f1cognite) [inline src/components/AssetSearch/AssetSearch.tsx]: Remove this?

File: src/components/AssetSearchForm/AssetSearchForm.tsx [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+          >
+            {search}
+          </Button>
+        </Form.Item>
+      )}
+    </Form>
+  );
+};
+
+const AssetSearchFormHOC = Form.create({
```
Comment (f1cognite) [inline src/components/AssetSearchForm/AssetSearchForm.tsx]: What does HOC stand for?

File: src/components/AssetSearch/AssetSearch.tsx (hunk 1 above)
Comment (mrtraser) [reply src/components/AssetSearch/AssetSearch.tsx]: Yep, I'll remove

File: src/components/AssetSearchForm/AssetSearchForm.tsx (hunk 2 above)
Comment (mrtraser) [reply src/components/AssetSearchForm/AssetSearchForm.tsx]: HOC provide form property with bunch of methods, which helps to grep data from form (such as getFieldDecorator, getFieldValue, setFieldsValue)

**Pull request merged**